config.c

```diff
@@ -120,140 +121,106 @@ struct pv_config_entry {
-	{ INT, "PV_LOG_MAXSIZE", PV_ENTRY | RUN_TIME, 0,
+	{ INT, "PV_LOG_LEVEL", PV | UPDATE | RUN, 0, .value.i = 0 },
+	{ BOOL, "PV_LOG_LOGGERS", PV | UPDATE, 0, .value.b = true },
+	{ INT, "PV_LOG_MAXSIZE", PV | UPDATE | RUN, 0,
 	  .value.i = LOG_MAXSIZE_DEF },
-	{ BOOL, "PV_LOG_PUSH", PV_ENTRY | RUN_TIME, 0, .value.b = true },
+	{ BOOL, "PV_LOG_PUSH", PV | UPDATE | RUN, 0, .value.b = true },
 	{ LOG_SERVER_OUTPUT_UPDATE_MASK, "PV_LOG_SERVER_OUTPUTS",
-	  PV_ENTRY | RUN_TIME, 0,
+	  PV | UPDATE | RUN, 0,
 	  .value.i = LOG_SERVER_OUTPUT_FILE_TREE | LOG_SERVER_OUTPUT_UPDATE },
-	{ STR, "PV_LOG_SINGLEFILE_TIMESTAMP_FORMAT", PV_ENTRY | RUN_TIME, 0,
+	{ STR, "PV_LOG_SINGLEFILE_TIMESTAMP_FORMAT", PV | UPDATE | RUN, 0,
 	  .value.s = NULL },
-	{ STR, "PV_LOG_STDOUT_TIMESTAMP_FORMAT", PV_ENTRY | RUN_TIME, 0,
+	{ STR, "PV_LOG_STDOUT_TIMESTAMP_FORMAT", PV | UPDATE | RUN, 0,
 	  .value.s = NULL },
-	{ INT, "PV_LXC_LOG_LEVEL", PV_ENTRY | UPDATE_TIME, 0, .value.i = 2 },
-	{ STR, "PV_NET_BRADDRESS4", PV_ENTRY | UPDATE_TIME, 0,
+	{ INT, "PV_LXC_LOG_LEVEL", PV | UPDATE, 0, .value.i = 2 },
+	{ STR, "PV_NET_BRADDRESS4", PV | UPDATE, 0,
 	  .value.s = NET_BRADDRESS4_DEF },
-	{ STR, "PV_NET_BRDEV", PV_ENTRY | UPDATE_TIME, 0,
-	  .value.s = NET_BRDEV_DEF },
-	{ STR, "PV_NET_BRMASK4", PV_ENTRY | UPDATE_TIME, 0,
-	  .value.s = NET_BRMASK4_DEF },
-	{ STR, "PV_POLICY", PV_ENTRY, 0, .value.s = NULL },
-	{ INT, "PV_REVISION_RETRIES", PV_ENTRY | RUN_TIME, 0, .value.i = 10 },
-	{ BOOL, "PV_SECUREBOOT_CHECKSUM", PV_ENTRY | POLICY, 0,
-	  .value.b = true },
-	{ BOOL, "PV_SECUREBOOT_HANDLERS", PV_ENTRY | POLICY, 0,
-	  .value.b = true },
-	{ SB_MODE, "PV_SECUREBOOT_MODE", PV_ENTRY | POLICY, 0,
-	  .value.i = SB_LENIENT },
-	{ STR, "PV_SECUREBOOT_OEM_CN_NAME", PV_ENTRY | POLICY, 0,
-	  .value.s = NULL },
-	{ STR, "PV_SECUREBOOT_OEM_TRUSTORE", PV_ENTRY | POLICY, 0,
+	{ STR, "PV_NET_BRDEV", PV | UPDATE, 0, .value.s = NET_BRDEV_DEF },
+	{ STR, "PV_NET_BRMASK4", PV | UPDATE, 0, .value.s = NET_BRMASK4_DEF },
+	{ STR, "PV_OEM_NAME", PV, 0, .value.s = NULL },
+	{ STR, "PV_POLICY", PV, 0, .value.s = NULL },
+	{ INT, "PV_REVISION_RETRIES", PV | UPDATE | RUN, 0, .value.i = 10 },
+	{ BOOL, "PV_SECUREBOOT_CHECKSUM", PV, 0, .value.b = true },
+	{ BOOL, "PV_SECUREBOOT_HANDLERS", PV, 0, .value.b = true },
+	{ SB_MODE, "PV_SECUREBOOT_MODE", PV, 0, .value.i = SB_LENIENT },
+	{ STR, "PV_SECUREBOOT_OEM_TRUSTORE", PV, 0,
 	  .value.s = SECUREBOOT_OEM_TRUSTSTORE_DEF },
-	{ STR, "PV_SECUREBOOT_TRUSTSTORE", PV_ENTRY | POLICY, 0,
+	{ STR, "PV_SECUREBOOT_TRUSTSTORE", PV, 0,
 	  .value.s = SECUREBOOT_TRUSTSTORE_DEF },
-	{ STR, "PV_STORAGE_DEVICE", PV_ENTRY | POLICY, 0, .value.s = NULL },
-	{ STR, "PV_STORAGE_FSTYPE", PV_ENTRY | POLICY, 0, .value.s = NULL },
-	{ BOOL, "PV_STORAGE_GC_KEEP_FACTORY", PV_ENTRY | RUN_TIME, 0,
+	{ STR, "PV_STORAGE_DEVICE", PV, 0, .value.s = NULL },
+	{ STR, "PV_STORAGE_FSTYPE", PV, 0, .value.s = NULL },
+	{ BOOL, "PV_STORAGE_GC_KEEP_FACTORY", PV | UPDATE | RUN, 0,
```

Comment:
Can we name UPDATE == OEM? or is that used elswhere where UPDATE makes more sense?

Reply:
UPDATE is OEM + BSP (trail.config), it's only a define to save characters in the table